service-api/src/main/java/greencity/dto/tipsandtricks/TitleTranslationVO.java

```diff
@@ -0,0 +1,15 @@
+package greencity.dto.tipsandtricks;
+
+import greencity.dto.translation.TranslationVO;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+@Data
+@EqualsAndHashCode
```

Comment:
Same here. Please, check what is included in Data annotaion and remove everuthing that is unnecessary